spec/polaris-management-service.yml

```diff
@@ -877,6 +878,7 @@ components:
         propertyName: storageType
         mapping:
           S3: "#/components/schemas/AwsStorageConfigInfo"
+          S3_COMPATIBLE: "#/components/schemas/S3CompatibleStorageConfigInfo"
```

Comment:
Do we need to make accordingly change to Polaris cli? cc @eric-maynard. I’m OK to do that in a follow-up.